extension/view/js/InPageTranslation.js

```diff
@@ -205,12 +206,16 @@ class InPageTranslation {
     addQualityClasses () {
         document.querySelectorAll("[x-bergamot-sentence-score]").forEach(el => {
             const sentenceScore = parseFloat(el.getAttribute("x-bergamot-sentence-score"));
-            el.classList.toggle("bad", sentenceScore < 0 && sentenceScore < Math.log(0.5));
+            if (sentenceScore > 0 || sentenceScore < this.QE_THRESHOLD) {
+                el.classList.toggle("bad", true);
+            }
         });
 
         document.querySelectorAll("[x-bergamot-word-score]").forEach(el => {
             const wordScore = parseFloat(el.getAttribute("x-bergamot-word-score"));
-            el.classList.toggle("bad", wordScore < 0 && wordScore < Math.log(0.5));
+            if (wordScore > 0 || wordScore < this.QE_THRESHOLD) {
```

Comment:
same goes for `wordScore`

Reply:
Are you referring to https://github.com/browsermt/bergamot-translator/pull/370#issuecomment-1058123399?